### ISSUE TITLE
RGB not shown on expression

### ISSUE THREAD

**Issue by QuasarX** [Scirra/Construct-3-bugs#3116]
## Problem description

rgb(100,100,100) work but I don't see in expressions.

![image](https://user-images.githubusercontent.com/13033837/60767858-50bc5780-a0c6-11e9-9703-ad276419e14e.png)

![image](https://user-images.githubusercontent.com/13033837/60767861-5f0a7380-a0c6-11e9-825e-1dc92837c957.png)

**Comment by shortercode** [comment]
`rgb` is a legacy expression, I cannot remember what version we depreciated it in but it was awhile go ( maybe like 2/3 beta cycles, I think it was around about when the drawing canvas and advanced random plugins got added ). You will see it has been removed from the [documentation as well](https://www.construct.net/en/make-games/manuals/construct-3/system-reference/system-expressions).

When expressions are depreciated they continue to work, as so not to break projects that rely on them. However, they will not appear in any options lists any longer.
